### DISCUSSION TITLE
オープンソースガイド

### DISCUSSION THREAD

**Discussion by kazzyfrog** [first-contributions-ja/first-contributions-ja.github.io#78] [90: 知識・知見の共有]
オープンソース特有の作法・文化などについて、つらつら書いていくスレッドです！🙌

**Comment by kazzyfrog** [comment]
## オープンソースプロジェクトの関係者
一般的なオープンソースプロジェクトは、下記のような種類の人々がいます:
﻿
- オーナー: 組織やリポジトリに対して管理上の所有権を持っている人(たち)もしくは組織
- メンテナー: ビジョンを推し進め、プロジェクトの組織的なマネジメントを行うコントリビューター(オーナーであることもある)
- コミッター: プロジェクトに積極的にプッシュ（コミット）するコントリビューター
- コントリビューター: プロジェクトに対して何かしらのコントリビュートをするすべての人
- コミュニティメンバー（ユーザー）: プロジェクトを使っている人々

基本的に、コントリビューターの中でも、繰り返し貢献してくれる人には、リポジトリのアクセス権限・役割が与えられていく。
自分たちは、現在メンテナー！このリポジトリの管理者。

参考：
- [オープンソース開発について](https://envader.plus/course/12/scenario/1139)
- [オープンソースプロジェクトの構造](https://opensource.guide/ja/how-to-contribute/#オープンソースプロジェクトの構造)
- [リポジトリのロール](https://docs.github.com/ja/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization)
- [オープンソースプロジェクトで使われる役割](https://opensource.guide/ja/leadership-and-governance/#オープンソースプロジェクトで使われる公式の役割の例はなんですか)

**Reply by pss-aileen** [reply]
この名称についてどこかで共有した方が、参加する人が理解しやすそう！
（みんな自分がどれにあたるかって初心者はわからない気がした。自分もそんなかんじ....！）
